.golangci.yml

```diff
@@ -88,3 +90,9 @@ issues:
     - linters:
         - govet
       text: "pointer bytes could be"
+
+    # This requires changes in Admiral
+    - linters:
+        - gocritic
+      text: "hugeParam: syncerConfig is heavy"
+      path: pkg/controllers/datastoresyncer/datastoresyncer.go
```

Comment:
The violation is reported due to the `New` function in _datastoresyncer.go_ and not the Admiral code:

```
pkg/controllers/datastoresyncer/datastoresyncer.go:51:10: hugeParam: syncerConfig is heavy (144 bytes); consider passing it by pointer (gocritic)
func New(syncerConfig broker.SyncerConfig, localCluster *types.SubmarinerCluster,
```

So we just need to change that function to take a pointer to clear the error.